paddle/majel/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cc_library(place SRCS place.cc)
-cc_test(place_test SRCS place_test.cc DEPS place)
+cc_test(place_test SRCS place_test.cc DEPS place glog gflags)
```

Comment:
I think this is the right way -- to require the specification of glog and gflags explicitly -- and could be the default way.